servers/graphql-kotlin-ktor-server/src/main/kotlin/com/expediagroup/graphql/server/ktor/KtorGraphQLRequestParser.kt

```diff
@@ -23,8 +23,9 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.databind.type.MapType
 import com.fasterxml.jackson.databind.type.TypeFactory
 import io.ktor.http.HttpMethod
-import io.ktor.server.request.ApplicationRequest
-import io.ktor.server.request.receiveText
+import io.ktor.server.request.*
```

Comment:
ktlint complains about the wildcard imports -> can you fix it?

Reply:
Yep, done in https://github.com/ExpediaGroup/graphql-kotlin/pull/1673/commits/a5f75e17d4a270b947aa52ddbee692ad5f66c05c